src/app/App.js

```diff
@@ -0,0 +1,11 @@
+import './App.css';
+import React from 'react';
+import CurrentFilmDescription from '../components/CurrentFilm/index'
```

Comment:
If you are using index file for import you can write like this `import CurrentFilmDescription from '../components/CurrentFilm'` and all exports from index file will be imported